src/App.js

```diff
@@ -1,27 +1,31 @@
-import logo from './logo.svg';
+import { Routes, Route } from 'react-router-dom';
+import { useDispatch } from 'react-redux';
+import { useEffect } from 'react';
+import { fetchItem } from './redux/Slice/ItemSlice';
+import NavigationBar from './components/Navigation';
 import './App.css';
+import Home from './components/HomePage';
+import Details from './components/ItemDetaile/details';
 
 function App() {
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(fetchItem());
+  }, [dispatch]);
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          {' '}
-          <code>src/App.js</code>
-          {' '}
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <div className="header">
+        <NavigationBar />
+      </div>
+      <div className="container">
+        <Routes>
+          <Route index element={<Home />} />
+          <Route path="home" element={<Home />} />
+          <Route path="details" element={<Details />} />
+          {/*
+          <Route path="*" element={<NotFound />} /> */}
+        </Routes>
+      </div>
```

Comment:
Please remove unused code for it makes your codebases look untidy.

Reply:
Okay, Well Done ✔